index.js

```diff
@@ -4,6 +4,22 @@
 module.exports = {
   name: 'ember-truncate',
   included: function(app) {
+    var current = this;
+
+    // If the addon has the _findHost() method (in ember-cli >= 2.7.0), we'll just
+    // use that.
+    if (typeof this._findHost === 'function') {
+      app = this._findHost();
+    }
+
+    // Otherwise, we'll use this implementation borrowed from the _findHost()
+    // method in ember-cli.
+    // Keep iterating upward until we don't have a grandparent.
+    // Has to do this grandparent check because at some point we hit the project.
+    do {
+      app = current.app || app;
+    } while (current.parent.parent && (current = current.parent));
```

Comment:
I'm wondering if it might be better to bump the major version and add a peer dependency of `ember-cli@>=2.7.0`, so we don't have to "reimplement the wheel" here. What do you think?